<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePostView</class>
 <widget class="QWidget" name="CreatePostView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Post</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>771</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="titleInput">
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; border: 1px solid #ccc; color:black;</string>
      </property>
      <property name="placeholderText">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolBar" name="formattingToolbar">
        <property name="styleSheet">
         <string notr="true">
       QToolBar {
           background: #f5f5f5;
           border-radius: 5px;
           padding: 5px;
           spacing: 5px;
       }
       QToolButton {
           background: transparent;
           border: 1px solid #ccc;
           border-radius: 3px;
           padding: 5px;
       }
       QToolButton:hover {
           background: #e0e0e0;
       }
       QToolButton:pressed {
           background: #d0d0d0;
       }
      </string>
        </property>
        <addaction name="boldAction"/>
        <addaction name="italicAction"/>
        <addaction name="underlineAction"/>
        <addaction name="strikeAction"/>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="fontSizeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">    font: 500 11pt &quot;Raleway&quot;;
    border: 2px solid rgb(85, 170, 255);  /* Border color */
    background-color: transparent;  /* Remove background */
    color: black;  /* Text color */</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 500 11pt &quot;Raleway&quot;; color:black;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="uploadImageButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; background-color: #0079D3; color: white;</string>
        </property>
        <property name="text">
         <string>UP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="bodyInput">
      <property name="styleSheet">
       <string notr="true">color:black;</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Raleway&quot;; padding: 10px; border-radius: 5px; background-color: #0079D3; color: white;</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="iconText">
    <string>B</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="iconText">
    <string>I</string>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="underlineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="iconText">
    <string>U</string>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="strikeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Strike</string>
   </property>
   <property name="iconText">
    <string>S</string>
   </property>
   <property name="toolTip">
    <string>Strikethrough</string>
   </property>
   <property name="font">
    <font>
     <family>Raleway</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
